react/components/utils/useDevice.tsx

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react'
```

Comment:
can use useMedia directly

Reply:
The behaviors are reused in both tabs and tab

Reply:
can use in both

Reply:
but, I cannot reuse the conditions and the two hook calls

Reply:
I would literary copy & paste the hook body on each component, instead of just `const { isMobile } = useDevice()`. (the same  for other components in the styleguide that may/will need this in the future)

Reply:
you just need const isMobile = useMedia({ maxWidth: '40rem' }, true)